swatch-contracts/ct/java/tests/BaseContractComponentTest.java

```diff
@@ -20,19 +20,22 @@
  */
 package tests;
 
-import api.ContractsWiremockService;
 import com.redhat.swatch.component.tests.api.ComponentTest;
 import com.redhat.swatch.component.tests.api.Quarkus;
 import com.redhat.swatch.component.tests.api.SwatchService;
 import com.redhat.swatch.component.tests.api.Wiremock;
 import org.junit.jupiter.api.Tag;
+import wiremock.ContractsWiremockService;
+import wiremock.OfferingWiremockService;
 
 @ComponentTest
 @Tag("component")
 @Tag("contracts")
 public class BaseContractComponentTest {
 
-  @Wiremock static ContractsWiremockService wiremock = new ContractsWiremockService();
+  @Wiremock static ContractsWiremockService contractsWiremock = new ContractsWiremockService();
```

Comment:
This should be Partner API wiremock service, right?

Reply:
Changed to  PartnerGatewayStubs. Done